owncloudApp/src/androidTest/java/com/owncloud/android/shares/data/OCShareDaoTest.kt

```diff
@@ -284,15 +284,51 @@ class OCShareDaoTest {
         assertEquals("userName", text2Shares[0].shareWith)
     }
 
+    @Test
+    fun updatePrivateShare() {
+        ocShareDao.insert(createDefaultPrivateShare())
+
+        ocShareDao.update(
+            createDefaultPrivateShare(permissions = 17)
+        )
+
+        val textShares = getValue(
+            ocShareDao.getSharesAsLiveData(
+                "/Texts/text1.txt", "admin@server", listOf(ShareType.USER.value)
+            )
+        )
+
+        assertThat(textShares, notNullValue())
+        assertEquals(1, textShares.size)
+        assertEquals(17, textShares[0].permissions)
+    }
+
+    @Test
+    fun deletePrivateShare() {
+        ocShareDao.insert(createDefaultPrivateShare())
+
+        ocShareDao.deleteShare(1)
```

Comment:
Similar here, how do we know that the Share created by 'createDefaultPrivarteShare' is the one with id '1'?

Reply:
Done

Reply:
Done